build-logic/convention/src/main/kotlin/GravatarComposeConventionPlugin.kt

```diff
@@ -0,0 +1,60 @@
+import com.android.build.gradle.LibraryExtension
+import com.gravatar.libs
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.apply
+import org.gradle.kotlin.dsl.configure
+import org.gradle.kotlin.dsl.withType
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+class GravatarComposeConventionPlugin : Plugin<Project> {
+    override fun apply(target: Project) {
+        with(target) {
+            apply(plugin = "io.github.takahirom.roborazzi")
+
+            extensions.configure<LibraryExtension> {
+                buildFeatures {
+                    compose = true
+                }
+                composeOptions {
+                    kotlinCompilerExtensionVersion =
+                        libs.findVersion("kotlinCompilerExtension").get().toString()
+                }
+                testOptions {
+                    unitTests {
+                        // For Roborazzi
+                        isIncludeAndroidResources = true
+                        all {
+                            // -Pscreenshot to filter screenshot tests
+                            it.useJUnit {
+                                if (project.hasProperty("screenshot")) {
+                                    includeCategories("com.gravatar.uitestutils.ScreenshotTests")
+                                } else {
+                                    excludeCategories("com.gravatar.uitestutils.ScreenshotTests")
+                                }
```

Comment:
Out of curiosity and I haven't tested it but do we have to have both "include" and "exclude" mechanisms? Aren't things "excluded" by default?

```suggestion
                                if (project.hasProperty("screenshot")) {
                                    includeCategories("com.gravatar.uitestutils.ScreenshotTests")
                                }
```